tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noImplicitAny": false,
     "removeComments": true,
     "preserveConstEnums": true,
+    "strictPropertyInitialization": false,
```

Comment:
@ranm8 , do you remember the reason for disabling this?

Reply:
Yep, this is because Harmony now introduces a new property to extensions (which he initializes) called `id` to get access to the extension name from the extension itself.

```ts
class TesterExtension {
  readonly id: string; // will be auto-initialized by Harmony
}
```